### PR TITLE
feat: Update the webui to work with the latest changes in 0.4

### PR DESCRIPTION
**Pull request by dignifiedquire**
As reported in #2334 most people are happy with it :)

### CODE REVIEW

File: core/corehttp/webui.go [hunk 1]
```diff
@@ -13,6 +13,7 @@ var WebUIPaths = []string{
 	"/ipfs/QmctngrQAt9fjpQUZr7Bx3BsXUcif52eZGTizWhvcShsjz",
 	"/ipfs/QmS2HL9v5YeKgQkkWMvs1EMnFtUowTEdFfSSeMT4pos1e6",
 	"/ipfs/QmR9MzChjp1MdFWik7NjEjqKQMzVmBkdK3dz14A6B5Cupm",
+	"/QmRyWyKWmphamkMRnJVjUTzSFSAAZowYP4rnbgnfMXC9Mr",
```
Comment (diasdavid) [inline core/corehttp/webui.go]: no '/ipfs" for this one?

Comment (dignifiedquire) [inline core/corehttp/webui.go]: no this one doesn't get one :P

File: core/corehttp/webui.go [hunk 2]
```diff
@@ -13,6 +13,7 @@ var WebUIPaths = []string{
 	"/ipfs/QmctngrQAt9fjpQUZr7Bx3BsXUcif52eZGTizWhvcShsjz",
 	"/ipfs/QmS2HL9v5YeKgQkkWMvs1EMnFtUowTEdFfSSeMT4pos1e6",
 	"/ipfs/QmR9MzChjp1MdFWik7NjEjqKQMzVmBkdK3dz14A6B5Cupm",
+	"ipfs/QmRyWyKWmphamkMRnJVjUTzSFSAAZowYP4rnbgnfMXC9Mr",
```
Comment (diasdavid) [inline core/corehttp/webui.go]: no '/' on this one?

**Comment by whyrusleeping** [comment]
Is this well pinned?

**Comment by dignifiedquire** [comment]
@whyrusleeping not sure (pinbot seems still slow/unreliable), @lgierth could you check this please?

**Pull request merged**